CMakeLists.txt

```diff
@@ -71,7 +62,7 @@ endmacro()
 # NO_CMAKE_PACKAGE_REGISTRY used to avoid finding the library downloaded in WORKSPACE B
 # when building workspace A.
 # This should only find a system installed yaml-cpp and thus the environment hook isn't needed.
-find_package(yaml-cpp 0.6 QUIET NO_CMAKE_PACKAGE_REGISTRY)
+find_package(yaml-cpp QUIET NO_CMAKE_PACKAGE_REGISTRY)
```

Comment:
Why was this removed but not updated? (just curious)

Reply:
Mostly because we don't do that in any of our other vendor packages.  Looking through all of the ones I see in ROS 2 right now, I can't find a single example where we specify the version when trying to find the system package.  So I thought I would be consistent here.